woocommerce/woocommerce-ios
# [Deeplink] Handle “deeplink” to `/mobile/`

## Description


We did not handle the `/mobile` subpath, because there was no specific route for it.

This commit adds a route which opens the My Store tab.

## Testing instructions


1. Install the app
2. Open woocommerce.com in mobile Safari
3. Scroll to the bottom and tap the `Mobile app` link
4. Click `Open` in the top bar on the web page, to "open in the woo app"
5. Observe that the app is opened and switches to the My Store tab.

## Screenshots



https://github.com/woocommerce/woocommerce-ios/assets/2472348/25d563b4-533f-4b13-9fc9-791555caff4f


---
